en/prepare/attributes-from.rst

```diff
@@ -79,18 +80,20 @@ The second case is usually used when there are additional relation information (
 
 In case of the many-to-one cardinality on many-to-many relations the Display Expression needs to be set on the linking table.
 
-External Resource (photo settings)
-..................................
+.. _attachment:
 
-The widget type "Attachement" is used to store the path to collected photos.
+Attachment (photo settings)
+...........................
+
+The widget type "Attachement" is used to store the path to collected photos, pdf and docs.
```

Comment:
```suggestion
The widget type "Attachement" is used with fields that store the path to files.

It can be used to:

- show and take photos
- or to show links to external files like pdfs or documents
```